.github/workflows/open-pull-request-for-icon-update.yml

```diff
@@ -38,5 +38,43 @@ jobs:
           branch-suffix: 'short-commit-hash'
           commit-message: 'sync & build of flight icons'
           title: 'Updated export of icons from Figma'
-          body: ''
+          body: |
+            ### :pushpin: Summary
+
+            This is an automated PR opened by the Hashibot-HDS using the GitHub Action [Sync & Build Icons](https://github.com/hashicorp/design-system/actions/workflows/open-pull-request-for-icon-update.yml).
+
+            ### :hammer_and_wrench: Detailed description
+
+            <!-- (Remove this comment) once done ------------
+
+            Add specific details for this PR here:
+            - which icons has been added?
+            - which icons have been changed, updated or deleted?
+
+            Make sure that you have followed the instructions described here:
+            https://github.com/hashicorp/design-system/blob/main/packages/flight-icons/CONTRIBUTING.md#updating-the-icons-in-flight-icons ("Updating the icons in flight-icons" + "Automated process" + "After you have run the scripts")
+
+            Make also that the "changeset" is added to the PR _before_ asking for review (and expecially don't merge to `main` if the changeset is missing).
```

Comment:
Could we take the template for a changeset that @heatherlarsen is developing and bake it into this template so that when someone opens an icons PR they could copy and paste it directly from here?

Reply:
yes, good idea. but let's do it for both the templates (generic PR and this one)